smurfs/src/components/SmurfForm.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import axios from 'axios'
+
+const SmurfForm = () => {
+    const [name, setName] = useState()
+    const [age, setAge] = useState()
+    const [height, setHeight] = useState()
+
+    const handleNameChange = (e) => {
```

Comment:
And then change this to `handlgeChange` and reuse it for every `<input>`
```suggestion
    const handleChange = e => {
        setSmurf({
            ...smurf,
            [ e.target.name ]: e.target.value
        });
    }
```